packages/main/src/plugin/appearance-init.ts

```diff
@@ -38,5 +40,22 @@ export class AppearanceInit {
     };
 
     this.configurationRegistry.registerConfigurations([appearanceConfiguration]);
+
+    this.configurationRegistry.onDidChangeConfiguration(async e => {
+      if (e.key === AppearanceSettings.SectionName + '.' + AppearanceSettings.Appearance) {
```

Comment:
```suggestion
      if (e.key === `{AppearanceSettings.SectionName}.${AppearanceSettings.Appearance}`) {
```
also maybe compute the key once as a constant instead of inside the if

```ts
const APPEARANCE_FULL_KEY  = `{AppearanceSettings.SectionName}.${AppearanceSettings.Appearance}`
...
...
...

 if (e.key === APPEARANCE_FULL_KEY) {
...
```

Reply:
Forgot to push a commit before PTO - pushed now. Uses a constant for both instances.